catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/fs/FileSystemUtils.java

```diff
@@ -41,12 +41,27 @@ public static Map<String, FileSystemProvider> getFileSystemProviders(String file
     ServiceLoader<FileSystemProvider> allFileSystemProviders =
         ServiceLoader.load(FileSystemProvider.class);
 
+    if (null == fileSystemProviders) {
+      // If the file system providers are not specified, return all file system providers.
+      Streams.stream(allFileSystemProviders.iterator())
+          .forEach(
+              fileSystemProvider -> {
+                if (resultMap.containsKey(fileSystemProvider.scheme())) {
+                  throw new UnsupportedOperationException(
+                      String.format(
+                          "File system provider: '%s' with scheme '%s' already exists in the provider list, "
+                              + "please make sure the file system provider scheme is unique.",
+                          fileSystemProvider.getClass().getName(), fileSystemProvider.scheme()));
+                }
+                resultMap.put(fileSystemProvider.scheme(), fileSystemProvider);
+              });
+      return resultMap;
```

Comment:
Do we also need to add HDFS and local FS provider here?

Reply:
```
ServiceLoader<FileSystemProvider> allFileSystemProviders =
        ServiceLoader.load(FileSystemProvider.class);
```
The code above will load HDFS and Local FS automatically.